test/test_computed_field.jl

```diff
@@ -212,9 +212,11 @@ function computation_including_boundaries(arch)
     model = NonhydrostaticModel(grid=grid)
 
     u, v, w = model.velocities
-    @. u.data = 1 + rand()
-    @. v.data = 2 + rand()
-    @. w.data = 3 + rand()
+    CUDA.@allowscalar begin
```

Comment:
Is there any way to do this without `@allowscalar`? We should be reducing where allowscalar appears in tests, not adding new tests with this.